tests/pytorch/nightly/resnet50-mp.libsonnet

```diff
@@ -118,5 +180,7 @@ local utils = import 'templates/utils.libsonnet';
     resnet50_MP + v3_8 + functional + timeouts.Hours(2),
     resnet50_gpu + common.Functional + v100 + timeouts.Hours(2),
     resnet50_gpu + common.Functional + v100x4 + timeouts.Hours(1),
+    common.PyTorchTest + resnet50_tpu_vm + v3_8 + functional_tpu_vm + timeouts.Hours(2),
```

Comment:
To be consistent with the other tests, you can do this:

`local resnet50_tpu_vm = common.PyTorchTest { ...`

`resnet50_tpu_vm + v3_8 + functional_tpu_vm + timeouts.Hours(2) + experimental.PyTorchTpuVmMixin`

Reply:
Good idea, switched